repo-infra/.kazelcfg.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
repo-infra should go into `vendor` dir; use `requires` in gopkg.toml to get it to vendor correctly